### PR TITLE
NEW: Improve Metadata Estimation

### PR DESCRIPTION
**Pull request by John-P**
- Add a better microns per-pixel (MPP) from objective power estimation function (y = 10 / x) which is continuous and self-inverse i.e. 40 → 0.25, 0.25 → 40, 0.5 → 20, 20 → 0.5, 0.2534 → 39.46, 39.46 → 0.2534 etc.
- Add a method to estimate objective power from MPP but snap to common values e.g. 20x, 40x etc.

### CODE REVIEW

File: tiatoolbox/utils/misc.py [hunk 1]
```diff
@@ -202,36 +202,60 @@ def get_luminosity_tissue_mask(img, threshold):
...
+        float: Microns per-pixel (MPP) approximation.
+
+    Raises:
+        ValueError
+    """
+    return 10 * (np.float(objective_power) ** -1)
+
+
+@np.vectorize
 def mpp2objective_power(mpp):
```
Comment (shaneahmed) [inline tiatoolbox/utils/misc.py:246]: I cannot understand this function. This function says mpp2objective but it's returning objectivepower2mpp?

**Review by shaneahmed — COMMENTED** [review]
Not sure about implementation. Please can you double check?

File: tiatoolbox/utils/misc.py (hunk 1 above)
Comment (John-P) [reply tiatoolbox/utils/misc.py:246]: It is a self-inverse function hence the inverse calls the other.

**Comment by John-P** [comment]
> Not sure about implementation. Please can you double check?

Can you clarify what you are unsure about?

**Review by shaneahmed — APPROVED** [review]

**Comment by shaneahmed** [comment]
Please can you add additional functions to [usage.rst](https://github.com/TIA-Lab/tiatoolbox/blob/develop/docs/usage.rst)?

**Comment by John-P** [comment]
Docs building at https://readthedocs.org/dashboard/tia-toolbox/version/feature-metadata-estimation/

**Review by shaneahmed — APPROVED** [review]

**Pull request merged**